### DISCUSSION TITLE
how do i customize sddm?

### DISCUSSION THREAD

**Discussion by dudenukm** [prasanthrangan/hyprdots#484] [General]
hey, Linux newbie here, I installed Hyprland a few weeks ago when I installed arch. Can you guide me how to configure sddm?

**Comment by kRHYME7** [comment]
Hi, can we move this to the Discussions? This seemed not related to configuration issues.  Someone will still be able to help in the Discussion panel. 
BTW, you could change your sddm wallpaper instead of using ```  ~/.config/hypr/scripts/sddmwall.sh ```. As I understood the script this fetches your current wallpaper and sends it sddm as your "lock screen ".

**Reply by Balssh** [reply]
Uhm, I didn't even knew we had this. Is the sddmwall supposed to be used by default as my sddm wallpaper is the same no matter the theme/wallpaper.

**Reply by kRHYME7** [reply]
Yup, I guess this is due to the privileges needed to set the wallpaper for sddm, that's how I interpret it when I read the script. I don't actually use sddm though. I just autologin and added hyprland on ./.zprofile.

**Comment by prasanthrangan** [comment]
> hey, Linux newbie here, I installed Hyprland a few weeks ago when I installed arch. Can you guide me how to configure sddm?

Hi, some parameters for sddm theme (colors, background, date format, font) can be configured in `/usr/share/sddm/themes/corners/theme.conf` file

if you want to modify the structure then you'll have to modify the qml files in 
`/usr/share/sddm/themes/corners/components`
